package.yaml

```diff
@@ -1,5 +1,5 @@
 name: aws-lambda-haskell-runtime
-version: 2.0.6
+version: 3.0.0
```

Comment:
I have bumped the version to 3.0.0 as the changes are very significant. Feel free to change that to whatever you guys find suitable.